pkg/dns/aws/dns.go

```diff
@@ -309,6 +310,15 @@ func (m *Manager) updateAlias(domain, zoneID, target, targetHostedZoneID, action
 		},
 	})
 	if err != nil {
+		if action == "DELETE" {
+			if aerr, ok := err.(awserr.Error); ok {
+				if aerr.Code() == route53.ErrCodeInvalidChangeBatch {
```

Comment:
does this code _always_ mean the delete was a no-op?

Reply:
I see what I did wrong now. I'll update and resubmit the PR.